_projects/SP19-AutoIndustries

```diff
@@ -0,0 +1,11 @@
+---
+layout: project
+title: Auto Industries
+desc: |
+ Simulator for grinding a car's bumper to help recovering patients in PT and introduce new concepts to newcomers.
+dl: https://drive.google.com/drive/folders/1EUd0VZfbfGQckiOmnWE-NUrCMM1aPte4?usp=sharing
```

Comment:
Please ensure this link is accessible for anyone with the link

Reply:
Should be fixed as of ~10 hours ago, let me know if it isn't.